extensions/shared/src/main/java/app/revanced/extension/youtube/settings/Settings.java

```diff
@@ -475,6 +475,11 @@ public class Settings extends BaseSettings {
     public static final IntegerSetting META_PANEL_BOTTOM_MARGIN = new IntegerSetting("revanced_shorts_meta_panel_bottom_margin", 32, true, parent(ENABLE_TIME_STAMP));
     public static final BooleanSetting HIDE_SHORTS_TOOLBAR = new BooleanSetting("revanced_hide_shorts_toolbar", FALSE, true);
     public static final BooleanSetting HIDE_SHORTS_NAVIGATION_BAR = new BooleanSetting("revanced_hide_shorts_navigation_bar", FALSE, true);
+    public static final BooleanSetting HOOK_MORE_BUTTON = new BooleanSetting("revanced_hook_more_button", TRUE);
+    public static final BooleanSetting COPY_URL_SHORT_TOOLBAR_MENU = new BooleanSetting("revanced_hook_more_button_option_copy_link", TRUE);
+    public static final BooleanSetting COPY_URL_WITH_TIMESTAMP_SHORT_TOOLBAR_MENU = new BooleanSetting("revanced_hook_more_button_option_copy_link_timestamp", TRUE);
+    public static final BooleanSetting DOWNLOAD_SHORT_TOOLBAR_MENU = new BooleanSetting("revanced_hook_more_button_option_download_short", TRUE);
+    public static final BooleanSetting OPEN_IN_NORMAL_PLAYER_SHORT_TOOLBAR_MENU = new BooleanSetting("revanced_hook_more_button_option_open_in_normal_player", TRUE);
```

Comment:
it's best to add the dependency of `ENABLE_OPEN_LINKS_DIRECTLY` here, but it's defined after this, so it throws an error.

adding the dependency to `revanced_prefs.xml` solves the issue, but breaks the searching algorithm

Reply:
This is not necessary since RVX specify the package name in the Intent before launchIntent